test/subscriptionSpec.js

```diff
@@ -126,11 +126,19 @@ describe('Subscriptions', function () {
         executionCondition: destinationTransfer.debits[0].execution_condition,
         noteToSelf: {
           source_transfer_ledger: sourceTransfer.ledger,
-          source_transfer_id: sourceId
+          source_transfer_id: sourceId,
+          source_transfer_amount: sourceTransfer.debits[0].amount
         }
       }, 'cf:0:')
 
     sinon.assert.calledOnce(fulfillSpy)
     sinon.assert.calledWith(fulfillSpy, sourceId, 'cf:0:')
+    sinon.assert.calledOnce(backendSpy)
+    sinon.assert.calledWith(backendSpy, {
+      source_ledger: sourceTransfer.ledger,
+      source_amount: sourceTransfer.debits[0].amount,
+      destination_ledger: destinationTransfer.ledger,
+      destination_amount: destinationTransfer.credits[0].amount
+    })
```

Comment:
I think it would be clearer to separate this into another unit test, rather than bundling all the things its supposed to do into one test